.github/scripts/macos/programs/test-alt-ergo.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -euo pipefail
+
+# This script tests .pkg installer creation, installation, and functionality
+
+REMOTE_USER="arozovyk"
+REMOTE_BRANCH="alt-ergo-mac-builds"
```

Comment:
Any specific reason not to use the upstream repo and a specific tag?

Reply:
Ah my bad, misread this, you're fetching a prebuilt alt-ergo binary sitting on a branch of this repo!